cli/run_util.go

```diff
@@ -1091,6 +1091,15 @@ func runWorkflowWithConfiguration(
 		}
 	}
 
+	// Trigger WillStartRun
+	buildRunStartModel := models.BuildRunStartModel{
```

Comment:
I was not sure to include any properties here so please let me know what do you think of this.

Reply:
Looks good thanks